admin_scripts/ubuntu/protect_settings.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ !  -f "/usr/bin/gnome-control-center.real" ]
+then
+    dpkg-divert --divert  /usr/bin/gnome-control-center --add /usr/bin/gnome-control-center.real
+    dpkg-statoverride superuser root 770 /usr/bin/gnome-control-center.real
```

Comment:
As before, I think you want `--update` here to make sure that the file actually gets changed and not just protected from `dpkg` changes.

Reply:
Also done.